notebooks/python_intro/05-functions-write.md

```diff
@@ -111,9 +111,10 @@ def multiply_it(x, multiplication_factor=2):
 
 print(multiply_it(3))
 print(multiply_it(3, 3))
+print(multiply_it(3, multiplication_factor=4))
 ```
 
-The order of the keyword arguments does not matter, but it is good practice to use the same ordering as the function's definition. Keyword arguments 
+The order of the keyword arguments (when specifying by name) does not matter, but it is good practice to use the same ordering as the function's definition. Keyword arguments 
```

Comment:
The order of  keyword arguments actually does matter when passing them by position, as we have always been doing up to this point in the notebooks, I think. Tried to clarify this a bit and added an extra line above with using a named argument, which can trigger you to explain that both are possible.